packages/core/src/sync-store/postgres/store.ts

```diff
@@ -350,7 +356,7 @@ export class PostgresSyncStore implements SyncStore {
       }
 
       if (batch.length > 0) {
-        yield batch.map((a) => a.childAddress);
+        yield batch.map((a) => checksumAddress(a.childAddress));
```

Comment:
Again just confirming my understanding - this is required because the `selectChildAddressExpression` returns lowercase addresses, right?

Reply:
yes